docs/user-guide/liveness/README.md

```diff
@@ -66,11 +66,13 @@ The [http-liveness.yaml](http-liveness.yaml) demonstrates the HTTP check.
       httpGet:
         path: /healthz
         port: 8080
+        httpHeaders:
+          X-Custom-Header: [ Awesome ]
```

Comment:
Sure we should support it, but should we support is as a multi-map (like this) or as a list of {key, value} (which feels  more natural to me and less "because that is how Go does it).